runtime/fundamentals/configuration.md

```diff
@@ -487,6 +487,30 @@ works as well:
 }
 ```
 
+## Exports
+
+The `exports` field in the `deno.json` file allows you to define which modules
+or paths are publicly accessible from your package. This is particularly useful
+for controlling the API surface of your package and ensuring that only the
+intended parts of your code are exposed to users.
+
+```json title="deno.json"
+{
+  "exports": {
+    "./module1": "./src/module1.ts",
+    "./module2": "./src/module2.ts",
+    "./utils/*": "./src/utils/*.ts"
```

Comment:
Are you sure that this syntax is supported in Deno? It is in Node, but I can't get it to work in Deno.

Reply:
not sure at all, if its not working will take it out